### PR TITLE
consider academic year config for program-years and ci reports

### PR DESCRIPTION
**Pull request by stopfstedt**
fixes https://github.com/ilios/frontend/issues/5866
fixes https://github.com/ilios/frontend/issues/5878

### CODE REVIEW

File: app/components/learnergroup-header.js [hunk 1]
```diff
@@ -1,51 +1,33 @@
...
-  title: [
-    validator('presence', true),
-    validator('length', {
-      min: 3,
-      max: 60
-    })
-  ]
-});
+@validatable
+export default class ProgramyearHeaderComponent extends Component {
```
Comment (jrjohnson) [inline app/components/learnergroup-header.js:8]: Should be LearnergroupHeaderComponent

File: app/routes/curriculum-inventory-reports.js [hunk 2]
```diff
@@ -14,9 +14,4 @@ export default class CurriculumInventoryReportsRoute extends Route {
     const store = this.store;
     return store.findAll('school');
   }
-
-  setupController(controller, model) {
-    super.setupController(controller, model);
-    controller.set('sortByTitle', ['title']);
```
Comment (jrjohnson) [inline app/routes/curriculum-inventory-reports.js:20]: Love getting rid of these. They always bothered me.

File: app/templates/curriculum-inventory-reports.hbs [hunk 3]
```diff
@@ -1,95 +1,12 @@
...
-          @remove={{action "removeCurriculumInventoryReport"}}
-          @setSortBy={{action (mut this.sortReportsBy)}}
-        />
-      {{else}}
-        <PulseLoader />
-      {{/if}}
-    </div>
-  </section>
-</section>
+<CurriculumInventoryReports
```
Comment (jrjohnson) [inline app/templates/curriculum-inventory-reports.hbs:2]: ❤️

**Review by jrjohnson — CHANGES_REQUESTED** [review]
One super minor thing. I'm also going back in forth over the structure of the years. It's repeated so many times I wonder if a component to display year that loaded the config internally would be better than repeating the config load and display logic in so many places. I'm ok with this being a future improvement though.

File: app/components/learnergroup-header.js (hunk 1 above)
Comment (stopfstedt) [reply app/components/learnergroup-header.js:8]: :heavy_check_mark:

**Comment by stopfstedt** [comment]
> I'm also going back in forth over the structure of the years. It's repeated so many times I wonder if a component to display year that loaded the config internally would be better than repeating the config load and display logic in so many places. I'm ok with this being a future improvement though.

we could make another pass over this, should be easier this time around since we can now grep for the app config value's name. i've been seeing some common functionality that could be repurposed for the academic-year drop downs. outside of that, the labeling seems to be all over the place.

**Review by jrjohnson — APPROVED** [review]

**Pull request merged**